src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -23,107 +22,305 @@ internal static class TrustedSignersCommand
+
+                    CommandOption configFile = authorCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    authorCommand.HelpOption(XPlatUtility.HelpOption);
 
-                CommandOption configFile = trustedSignersCmd.Option(
+                    CommandOption verbosity = authorCommand.VerbosityOption();
+
+                    CommandArgument name = authorCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = authorCommand.Argument("<PACKAGE>",
+                                               Strings.TrustLocalSignedNupkgPath);
+
+                    authorCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Author, algorithm: null, allowUntrustedRootOption.HasValue(), owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: package.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("repository", (repositoryCommand) =>
+                {
+                    repositoryCommand.Description = Strings.TrustRepositoryCommandDescription;
+
+                    CommandOption allowUntrustedRootOption = repositoryCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = repositoryCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    repositoryCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption owners = repositoryCommand.Option(
+                        "--owners",
+                        Strings.TrustCommandOwners,
+                        CommandOptionType.MultipleValue);
+
+                    CommandOption verbosity = repositoryCommand.VerbosityOption();
+
+                    CommandArgument name = repositoryCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = repositoryCommand.Argument("<PACKAGE>",
+                                               Strings.TrustLocalSignedNupkgPath);
+
+                    repositoryCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Repository, algorithm: null, allowUntrustedRootOption.HasValue(), owners: owners, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: package.Value);
```

Comment:
If we change `packagePath` to accept multiple values, then we have to pass `packagePath.Values`.